assets/src/block-editor/components/with-media-library-notice.js

```diff
@@ -21,6 +26,10 @@ const { wp } = window;
  * @return {Function} The wrapped component.
  */
 export default ( InitialMediaUpload, minImageDimensions ) => {
+	if ( ! isObject( InitialMediaUpload ) ) {
```

Comment:
I'm wondering if we should update the JSDoc above to specify that this may be: `{Function|Object}`?